src/main/java/ca/warp7/frc2020/commands/FeedAutoCommand.java

```diff
@@ -0,0 +1,49 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package ca.warp7.frc2020.commands;
+
+import ca.warp7.frc2020.Constants;
+import ca.warp7.frc2020.subsystems.Feeder;
+import ca.warp7.frc2020.subsystems.Flywheel;
+import ca.warp7.frc2020.subsystems.Hopper;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class FeedAutoCommand extends CommandBase {
```

Comment:
can you move this to the `ca.warp7.frc2020.auton.commands` package, since this command is not used in teleop?